tree/tree/inc/TTree.h

```diff
@@ -216,8 +221,11 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
 
    // TTree status bits
    enum {
-      kForceRead   = BIT(11),
-      kCircular    = BIT(12)
+      kForceRead = BIT(11),
+      kCircular = BIT(12),
+      kFlushAtCluster = BIT(13) // If set, the branch's buffers will grow until an event cluster boundary is hit,
```

Comment:
This is an odd name.  By 'definition' the current behavior is to (auto) flush at the cluster boundaries ....